tests/system/action/motion_workflow/test_import.py

```diff
@@ -39,7 +39,13 @@ def test_import_simple_case(self) -> None:
                 "name": "test_Xcdfgee",
                 "meeting_id": 42,
                 "first_state_name": "begin",
-                "states": [self.get_state("begin", [], [])],
+                "states": [
+                    {
+                        **self.get_state("begin", [], []),
+                        "set_workflow_timestamp": True,
+                        "allow_motion_forwarding": True,
```

Comment:
Add these to `get_state` instead of only this payload

Reply:
Okay, I have added the two options to `get_state`.